config/environments/development.rb

```diff
@@ -46,7 +46,7 @@
   # config.i18n.raise_on_missing_translations = true
 
   # Annotate rendered view with file names.
-  # config.action_view.annotate_rendered_view_with_filenames = true
+  config.action_view.annotate_rendered_view_with_filenames = true
```

Comment:
Not a blocker, but would be nice to call out why we're enabling this

Reply:
Good point! I left it uncommented as part of the upgrade suggestion, but I can definitely comment it out as I can see it become too noisy when debugging. The idea was to make it easier for developers to debug by adding comments in HTML to show which view template was responsible for rendering a specific section of a page. 

But I could see that becoming annoying/cluttered. So I'll uncomment this now and I'll also deal with the merge conflict that I can see that came about!


Thanks Becca for spotting this :)

Reply:
I think you can leave it in and just have this comment as the explanation :)